src/test/java/org/folio/rest/api/ReferenceTablesTest.java

```diff
@@ -703,6 +706,28 @@ public void statisticalCodesAndCodeTypesBasicCrud()
 
   }
 
+  @Test
+  public void instanceRelationshipTypesLoaded() throws Exception {
+    Response searchResponse = getReferenceRecords(instanceRelationshipTypesUrl(""));
+    validateNumberOfReferenceRecords("instance-relationship types",
+      searchResponse, 4, 200);
+  }
+
+  @Test
+  public void instanceRelationshipTypesCrud() throws Exception {
```

Comment:
What does this test check?

Reply:
@marcjohnson-kint, it is basic CRUD operations for `/instance-relationship-types`  endpoint, for some reason there was no test present for this endpoint.

Reply:
Is this how the other tests for reference records are structured?

Reply:
@marc-johnson yes, commonly we have one test to check that reference data were loaded to DB and one test to check CRUD operations.

Reply:
Ok, if these are the established patterns for this area, then any feedback I might have is likely not relevant for this pull request.

(I think that records created during tests should be deleted prior to the next test, in order to improve the reliability and specificity of tests)